app/default_prefs.py

```diff
@@ -517,4 +518,8 @@ def numberTest(str, expectRegs):
       "backgroundIndexes": [231, 229, 14, 221,   255, 254, 253, 225],
     },
   },
+  'userData': {
+    'homePath': os.path.expanduser('~/.ci_edit'),
+    'historyPath': os.path.join(os.path.expanduser('~/.ci_edit'), 'history.dat'),
+  },
```

Comment:
Still need to work on bookmarks later on but I'm still not really sure what that does yet LOL